sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2562,17 +2562,34 @@ private static Throwable mapException(Throwable exception) {
 
         final ServiceBusManagementErrorException managementError = ((ServiceBusManagementErrorException) exception);
         final ServiceBusManagementError error = managementError.getValue();
-        switch (error.getCode()) {
+        final HttpResponse errorHttpResponse = managementError.getResponse();
+
+        Integer httpStatusCode = null;
```

Comment:
These can be final. Also to use primitive `int` because the statusCode will never be null.
```java
final int statusCode = error != null && error.getCode() != null
    ? error.getCode()
    : errorResponse.getStatusCode();
final int errorDetail = error != null && error.getDetail() != null
    ? error.getDetail()
    : errorResponse.getMessage();
```

Reply:
done